lib/python/picongpu/pypicongpu/output/binning.py

```diff
@@ -73,19 +74,36 @@ def translate_to_cpp_type(return_type):
     raise ValueError(f"Cannot translate {return_type=} to a C++ type.")
 
 
+def translate_to_cpp_unitDimVec(unit_dimension):
+    if unit_dimension == None:
```

Comment:
```suggestion
    if unit_dimension is None:
```
See e.g. https://stackoverflow.com/questions/3257919/what-is-the-difference-between-is-none-and-none